README.md

```diff
@@ -1 +1,96 @@
-# Lists-of-what-to-do
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)<br>
+- [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+💻 Getting Started<br>
+-Setup<br>
+-Prerequisites<br>
+- [👥 Authors](#authors)<br>
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [TodoList Project ] <a name="about-project"></a>
+
+
+*[Todo list project]* It's an application wich helps to make a list of activities that someone needs to accomplish within a day
+it includes :  <br>
+-List descriptions <br>
+-Marking lists as done on the list <br>
+-Removing lists that mark   ed as done <br>
+🛠 Built With<br>
+**Key Features**<br>
+simple Todo list Application, built with html5 & css3 and Javascript including linter configurattion.<br>
+
+[Set up linters]<br>
+[Add Lists on UI]<br>
+[Mark lists as done]<br>
+[Add Remove done lits]<br>
+**Tech Stack**<br>
+This TodoList
+project, was built with html5 and css3
+
```

Comment:
- [ ] Great work adding the sections 👏 . However, this looks a little bit clustered. Please use the proper markup to make it look professional 👍  

## View:
![image](https://user-images.githubusercontent.com/50990925/221120453-eb8cb06f-6e54-4a2b-8829-9d453c3bb3f3.png)